src/rtctools_heat_network/electricity_physics_mixin.py

```diff
@@ -0,0 +1,262 @@
+import logging
+
+import numpy as np
+
+from rtctools.optimization.collocated_integrated_optimization_problem import (
+    CollocatedIntegratedOptimizationProblem,
+)
+from rtctools.optimization.timeseries import Timeseries
+
+
+from .base_component_type_mixin import BaseComponentTypeMixin
+
+
+logger = logging.getLogger("rtctools_heat_network")
+
+
+class ElectricityPhysicsMixin(BaseComponentTypeMixin, CollocatedIntegratedOptimizationProblem):
```

Comment:
Docstring for class since it is a new class? I am not sure if it will add value though

Reply:
simple one added for now